### PR TITLE
Fix misspelling

### PR DESCRIPTION
**Pull request by tmcw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -437,7 +437,7 @@ last('hello');   // 'o'
 
 ## lens
 
-Create a lens to access a datastructur.
+Create a lens to access a datastructure.
```
Comment (bkendall) [inline README.md]: good catch - however, "data structure" is two words.

Comment (tjmehta) [inline README.md]: yup data structure

**Comment by stoeffel** [comment]
:articulated_lorry:

**Comment by bkendall** [comment]
:+1:

**Comment by tjmehta** [comment]
:+1:

**Pull request merged**